project_classification/project_classification_view.xml

```diff
@@ -46,10 +48,10 @@
       <field name="model">project.project</field>
       <field name="inherit_id" ref="project.edit_project" />
       <field name="arch" type="xml">
-        <xpath expr="/form/sheet/notebook/page[@string='Other Info']/group/group[3]/field[@name='parent_id']" position="replace">
-          <field name="classification_id" widget="selection" on_change="onchange_classification_id(classification_id)"/>
+        <xpath expr="/form/sheet/notebook/page[@string='Other Info']//field[@name='parent_id']" position="replace">
+          <field name="classification_id" required="1" widget="selection" on_change="onchange_classification_id(classification_id)"/>
```

Comment:
The best way to go would be to create a classification name something like: "Old project to classify" that is created only if projects are already present in database.

It might be needed to create the column already in pre_init_hook in order to populate it before adding the constraint.